.github/CODEOWNERS

```diff
@@ -1522,6 +1522,30 @@ x-pack/plugins/security_solution/common/api/entity_analytics @elastic/security-e
 /x-pack/test_serverless/functional/test_suites/security/ftr/cloud_security_posture/ @elastic/kibana-cloud-security-posture
 /x-pack/test_serverless/functional/test_suites/security/config.cloud_security_posture.ts @elastic/kibana-cloud-security-posture
 /x-pack/test_serverless/api_integration/test_suites/security/cloud_security_posture/ @elastic/kibana-cloud-security-posture
+/x-pack/plugins/fleet/public/components/agent_enrollment_flyout/steps/install_cloud_formation_managed_agent_step.tsx @elastic/fleet @elastic/kibana-cloud-security-posture
```

Comment:
WDYT to move the csp related files to one folder instead of listing individual files? It will be hard to remember to add any new files to this list later.

Reply:
yes, I think it makes sense to group them rather than have individual files. Will group them

Reply:
@juliaElastic moved things around to group `cloud_security_posture` related code together